### PR TITLE
Fix link to resources on submitResource page

### LINKED ISSUE #41: Fix link to /resources on "submit resource" page
Not sure if we're missing an import of react-router-dom or something, but the link at https://cb-react-concept.netlify.com/submit-resource appears broken.

Guessing this'll be a relatively straightforward fix, but haven't dug into this closely.

### PR DESCRIPTION
**Pull request by havenchyk**
fixes #41 

Formatting is done by prettier, not me

Kapture 2019-10-29 at 12 43 56

### CODE REVIEW

File: src/components/Resources/submitResource.js [hunk 1]
```diff
@@ -104,162 +119,180 @@ const SubmitResource = () => {
...
-            onChange={handleChange('title')}
-          />
+      <Grid item lg={9}>
+        <div className={clsx(classes.root, classes.breadcrumb)}>
+          <Paper elevation={0} className={classes.paper}>
+            <Breadcrumbs
+              separator={<NavigateNextIcon fontSize="small" />}
+              aria-label="breadcrumb"
+            >
+              <MuiLink color="inherit" component={Link} to="/resources">
```
Comment (havenchyk) [inline src/components/Resources/submitResource.js]: actual change

**Comment by lpatmo** [comment]
@havenchyk Thank you for taking a look at this! I tested out a simpler way to solve this:

+ `import { Link } from "react-router-dom";`
-  Link,

Could you try that out? 

Also, sorry for all the prettier-added diffs -- I merged that in last night; thought it would be less loud.

**Comment by lpatmo** [comment]
PS Thank you for the gif/video!

**Comment by havenchyk** [comment]
@lpatmo I used such over complicated solution only because of styling, so maybe it’s useless and just link from react-router is enough, I’ll rebase and change it, thanks

**Comment by havenchyk** [comment]
@lpatmo fixed, please take a look once again

**Comment by lpatmo** [comment]
@havenchyk Looks great! Thank you for making the change! :)

**Pull request merged**